Indico.RPAActivities/Indico.RPAActivities.Activities/Indico.RPAActivities.Activities.csproj

```diff
@@ -10,6 +10,7 @@
         <DocumentationFile>bin\$(Configuration)\Indico.RPAActivities.Activities.xml</DocumentationFile>
     </PropertyGroup>
     <ItemGroup>
+      <PackageReference Include="IndicoClient" Version="2.4.1" />
```

Comment:
since we didn't have references here before to this, we probably don't need to add one now